<?xml version="1.0" encoding="utf-8"?>

<launch>
  <node pkg="calibration" type="calibration_node" name="dynamic_calibration" output="screen"/>
  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen"/>
</launch>
